validation/near_global_lat_lon/near_global_quarter_degree.jl

```diff
@@ -6,10 +6,12 @@ using Oceananigans
 using Oceananigans.Units
 
 using Oceananigans.Fields: interpolate, Field
+using Oceananigans.Advection: VelocityStencil
 using Oceananigans.Architectures: arch_array
 using Oceananigans.Coriolis: HydrostaticSphericalCoriolis
 using Oceananigans.BoundaryConditions
 using Oceananigans.ImmersedBoundaries: ImmersedBoundaryGrid, GridFittedBottom, inactive_node, peripheral_node
+using Oceananigans.TurbulenceClosures: HorizontalDivergenceFormulation
```

Comment:
Should we export this from the top level?

Reply:
yeah let's do that